db/migrate/20200521150213_create_storage_services.rb

```diff
@@ -0,0 +1,18 @@
+class CreateStorageServices < ActiveRecord::Migration[5.1]
+  def change
+    create_table :storage_services do |t|
+      t.string :name
+      t.string :description
+      t.bigint :project_id
+      t.bigint :profile_id
```

Comment:
These look like foreign keys to other models, but we don't have project/profile tables.  Are these coming from the native provider?  If so these are badly named